### PR TITLE
feat: Query result display escaped newline

### LINKED ISSUE #186: Query results show escaped newline character
Currently, only the explain statement displays the escaped newline character. This results in a query string result not being displayed correctly, especially for functions like `json_pretty` that almost always have multiple lines of results.
For example:
```sql
root@127.0.0.1:8000/default> select json_pretty(parse_json('[1,2,[4,5,6,{"k":"v"}]]'));


SELECT
  json_pretty(parse_json('[1,2,[4,5,6,{"k":"v"}]]'))

┌──────────────────────────────────────────────────────────────────────────────────┐
│                json_pretty(parse_json('[1,2,[4,5,6,{"k":"v"}]]'))                │
│                                      String                                      │
├──────────────────────────────────────────────────────────────────────────────────┤
│ [\n  1,\n  2,\n  [\n    4,\n    5,\n    6,\n    {\n      "k": "v"\n    }\n  ]\n] │
└──────────────────────────────────────────────────────────────────────────────────┘
1 row result in 0.067 sec. Processed 1 rows, 1B (14.89 rows/s, 14B/s)
```

### PR DESCRIPTION
**Pull request by b41sh**
- Add `replace_newline` setting config to display no escaped newline as need
- Fix the `KeyWordCompleter` panic
- Fix nodejs binding Clippy error

Close #186

### CODE REVIEW

**Comment by sundy-li** [comment]
How about adding a new config named `replace_newline`?  Sometimes we need this config if the string result had '\n' which breaks the box style.

**Comment by everpcpc** [comment]
I think box display should always escape newline characters. Maybe we could introduce expanded display for repl.

**Comment by b41sh** [comment]
> I think box display should always escape newline characters. Maybe we could introduce expanded display for repl.

I've added a `replace_newline` setting config that displays escape newline characters by default and allows user to not escape newline as needed.

File: cli/src/session.rs [hunk 1]
```diff
@@ -429,15 +422,3 @@ impl From<&str> for QueryKind {
         }
     }
 }
-
-fn replace_newline_in_box_display(query: &str) -> bool {
-    let mut tz = Tokenizer::new(query);
-    match tz.next() {
-        Some(Ok(t)) => match t.kind {
-            TokenKind::EXPLAIN => false,
```
Comment (sundy-li) [inline cli/src/session.rs:437]: for `EXPLAIN` query, we should not respect this setting.

**Review by sundy-li — APPROVED** [review]

**Pull request merged**